tools/generate-esm-test.ts

```diff
@@ -0,0 +1,61 @@
+import fs from "fs";
+import { esmTestPort } from "../tests/helpers";
+
+const nodeVersion = process.versions.node.split(".").shift();
+const tsconfigBase = nodeVersion === "15" ? "14" : nodeVersion;
+
+const packageJson = `
+{
+  "name": "express-zod-api-esm-test",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "NODE_OPTIONS=\\"--loader ts-node/esm --es-module-specifier-resolution=node\\" node quick-start.ts"
```

Comment:
need to change all imports to complete filenames in order to get rid of this option
`--es-module-specifier-resolution`

Reply:
fixed in 8cecaef